bpf/bpf_lxc.c

```diff
@@ -50,6 +50,20 @@ static __always_inline bool redirect_to_proxy(int verdict, __u8 dir)
 }
 #endif
 
+#ifdef ENABLE_CUSTOM_CALLS
+static __always_inline void
+encode_custom_prog_meta(struct __ctx_buff *ctx, __u32 ret, int direction,
+			int identity)
```

Comment:
nit: `identity` seems to reflect the "local" endpoint (eg source for `direction == CUSTOM_CALLS_IDX_INGRESS` and destination for `direction == CUSTOM_CALLS_IDX_EGRESS`). Perhaps this should be indicated somewhere, either via the variable name or a comment?

Reply:
True, some polishing required on that side. I'll address it.